lerna.json

```diff
@@ -1,5 +1,4 @@
 {
-  "lerna": "3.13.4",
```

Comment:
why do we need to remove this ?

Reply:
It's deprecated from v3 and duplicated with the one in `package.json`.
See: https://github.com/lerna/lerna/pull/1225